ImperatorToCK3Tests/MapperTests/ReligionMapper/ReligionMappingTests.cpp

```diff
@@ -1,49 +1,145 @@
-#include "../../ImperatorToCK3/Source/Mappers/ReligionMapper/ReligionMapping.h"
+#include "Mappers/ReligionMapper/ReligionMapping.h"
+#include "Mappers/RegionMapper/CK3RegionMapper.h"
+#include "Mappers/RegionMapper/ImperatorRegionMapper.h"
 #include "gtest/gtest.h"
 #include <sstream>
 
 
-TEST(Mappers_ReligionMappingTests, ck3ReligionDefaultsToBlank)
+
+TEST(Mappers_ReligionMappingTests, regularMatchOnSimpleReligion)
 {
 	std::stringstream input;
-	input << "= {}";
+	input << "ck3 = flemish imp = dutch";
+	const mappers::ReligionMapping theMapping(input);
+
+	ASSERT_EQ("flemish", *theMapping.religionMatch("dutch", 56, 445));
```

Comment:
Use 0 for the provinces in the tests where they don't matter, to avoid a distracting detail.